### PR TITLE
feat/concat-string

### PR DESCRIPTION
**Pull request by gijsvanwichen**

### CODE REVIEW

File: functions/concat-string/1.0/function.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    "meta": {
+      "type": "Output",
+      "output": {
+        "type": "Text"
+      }
+    },
+    "name": "result",
+    "label": "Result",
+    "info": "The result as the concatenated string"
+  }],
```
Comment (thomasvanophem) [inline functions/concat-string/1.0/function.json:43]: I think we should make `left` and `right` required by adding 
```
"validations": {
  "required": true
}
```

File: functions/concat-string/1.0/function.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
+{
+  "description": "Concatenates two strings with an optional separator to combine two values into one string (eg, fullname is firstname + space + lastname)",
+  "label": "Concat String",
+  "category": "Misc",
+  "icon": "CreateIcon",
```
Comment (thomasvanophem) [inline functions/concat-string/1.0/function.json:5]: We need a different icon for this function

**Review by thomasvanophem — DISMISSED** [review]

File: functions/concat-string/1.0/index.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+const concatString = async ({ left, right, separator }) => {
+  return {
+    result: `${left}${separator}${right}`,
```
Comment (justin-betty) [inline functions/concat-string/1.0/index.js:3]: Can you use the javascript concat function

**Comment by justin-betty** [comment]
Please add tests

File: functions/say-hello/1.0/function.json [hunk 4]
```diff
@@ -2,7 +2,10 @@
   "label": "Say Hello",
```
Comment (justin-betty) [inline functions/say-hello/1.0/function.json:2]: This file can be removed

**Pull request merged**